src/aws/flb_aws_util.c

```diff
@@ -888,7 +889,8 @@ flb_sds_t flb_get_s3_key(const char *format, time_t time, const char *tag,
     tmp_key = NULL;
     flb_free(random_alphanumeric);
 
-    if (!gmtime_r(&time, &gmt)) {
+    gmt_tm = gmtime_r(&time, &gmt);
+    if (!gmt_tm) {
         flb_error("[s3_key] Failed to create timestamp.");
         goto error;
     }
```

Comment:
What is the point of this code change? I guess it follows convention better maybe? Not sure

Reply:
I advocate for removing all unnecessary code changes in this pr.